vehicle/mkvii/software/dashboard/dashboard.c

```diff
@@ -0,0 +1,207 @@
+/*
+    Header:
+    Contains code for the dashboard board
+    Function: Responsible for BMS and IMD LED Indicators - DONE
+                -listening for CAN messages and controlling LED appropriately
+            : Start button + LED - DONE
+                - based on input off of the physical start button before
+   entering RTD : Interface with LED Bars Boards - TODO
```

Comment:
LED Bars board is a comment I left when I wrote this code my first year :) and is no longer relevant to our team. Can you update this documentation comment to reflect the current firmware?